src/interface/blas3/backend/backend.hpp

```diff
@@ -22,15 +22,15 @@
  *  @filename backend.hpp
  *
  **************************************************************************/
-#if RCAR
+#ifdef RCAR
```

Comment:
```suggestion
#if defined RCAR
```
For consistency. Or better yet:
```suggestion
#if defined(RCAR)
```